src/gui/src/gui.cpp

```diff
@@ -1441,6 +1441,9 @@ Descriptor::Properties Selected::getProperties() const
   odb::Rect bbox;
   if (getBBox(bbox)) {
     props.push_back({"BBox", bbox});
+    // convenience; the user may want to know the dimensions
+    props.push_back({"Width", std::string(Descriptor::Property::convert_dbu(bbox.dx(), false))});
+    props.push_back({"Height", std::string(Descriptor::Property::convert_dbu(bbox.dy(), false))});
```

Comment:
Are the properties not scoped by the type they refer to?

Reply:
I am out of my depth here... I just hacked things until I saw something convenient in the GUI....

It is probably easier for someone skilled.in the art to make the two line change than to train me to do it.

Should I close this PR?

Reply:
I think he is just referring to the string "Width" versus "BBox Width" or the like.  We can probably work it out in this PR.

Reply:
could I append width/height to BBox or separate line?

I am confused about how Tcl enters into this.

Reply:
There are GUI commands in tcl that can access the properties.  @gadfort implemented it so he is best to answer.

Reply:
There is no scoping, i was just suggesting renaming the text from "Width" to "BBox width" (same for height) to avoid property name collisions. Shouldnt require much effort.

Reply:
OIC